src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs

```diff
@@ -230,6 +230,23 @@ private string MapPathWithParameters(string path, IEnumerable<OpenApiParameter>
             return newPath;
         }
 
+        private string MapBasePath(IList<OpenApiServer> servers)
+        {
+            if (servers == null || servers.Count == 0)
+            {
+                return null;
+            }
+
+            OpenApiServer server = servers.First();
+            Uri uriResult;
+            if (Uri.TryCreate(server.Url, UriKind.RelativeOrAbsolute, out uriResult))
+            {
+
+                return uriResult.IsAbsoluteUri ? uriResult.AbsolutePath : uriResult.ToString();
+            }
+            return null;
```

Comment:
Is this correct?

Reply:
Yes, if the Uril.TryCreate is false then a string.Empty should be returned. Comments?